### PR TITLE
Package

### PR DESCRIPTION
**Pull request by steveoni**
This pull request implements automatic building as specified in #27

### CODE REVIEW

File: src/copy.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+const path = require('path');
+const execSync = require('child_process').execSync
+
+
+const fileToCopy = {
+  'Metadata': 'import "./Metadata.css";',
+  'ProgressBar': 'import "./ProgressBar.css";',
+  'TablePreview':  undefined,
+  'TableSchema': 'import "./TableSchema.css";',
+  'Upload': undefined
```
Comment (cotts) [inline src/copy.js:11]: don't need to use quotes inside the object keys

File: src/copy.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+
+for(let keys in fileToCopy){
+
+  const fileContent = fs.readFileSync(
+    `${__dirname}/components/${keys}/index.jsx`
+    )
+    .toString()
+    .split('\n');
+  if (typeof fileToCopy[keys] != 'undefined') {
```
Comment (cotts) [inline src/copy.js:22]: Using  `(typeof fileToCopy[keys])` that should work too

**Review by cotts — CHANGES_REQUESTED** [review]
Awaiting review

File: src/copy.js (hunk 1 above)
Comment (steveoni) [reply src/copy.js:11]: that's true

**Pull request merged**